spec/helpers/navigation_helper_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "spec_helper"
+describe NavigationHelper, type: :helper do
+  describe "#navigation_links_internal" do
```

Comment:
These test are good to capture the logic well. I do think there's an opportunity to make them a fair bit more concise though:

```
    it "returns a list of internal links" do
      expect(navigation_links_internal).to include(a_hash_including(text: "Manuals", href: manuals_path))
    end

    it "toggles a what's new date based on whether it is before or after November 2023" do
      whats_new = a_hash_including(text: "What's new", href: whats_new_path)

      travel_to(Date.new(2023, 10, 31)) do
        expect(navigation_links_internal).to include(whats_new)
      end

      travel_to(Date.new(2023, 11, 1)) do
        expect(navigation_links_internal).not_to include(whats_new)
      end
    end

    it "sets the link to the current page as active" do
      request.path = manuals_path
      expect(navigation_links_internal).to include(a_hash_including(text: "Manuals", active: true))
    end
```